packages/common/services/logger.service.ts

```diff
@@ -144,6 +156,7 @@ export class Logger implements LoggerService {
     color: (message: string) => string,
     context = '',
     isTimeDiffEnabled?: boolean,
+    isError?: boolean,
```

Comment:
```suggestion
    writeStreamType?: 'stdout' : 'stderr',
```
What about changing the boolean to a union of write stream types (as shown in the suggestion)? It could fallback to 'stdout' by default

Reply:
Seems more appropriate!
I applied this review please checkout